core/templates/dev/head/editor/RouterServices.js

```diff
@@ -118,6 +124,18 @@ oppia.factory('routerService', [
       }
     };
 
+    var _actuallyNavigate = function(pathType, newStateName) {
+      if (pathType != 'preview' && pathType != 'gui') {
```

Comment:
MAIN_TAB is currently set as 'main' rather than 'gui'.  Could it be defined as 'gui' instead?